cranelift/codegen/src/isa/aarch64/lower.rs

```diff
@@ -381,28 +420,6 @@ fn put_input_in_rse<C: LowerCtx<I = Inst>>(
             };
             return ResultRSE::RegExtend(reg, extendop);
         }
-
-        // Is this a zero-extend or sign-extend and can we handle that with a register-mode operator?
-        if op == Opcode::Uextend || op == Opcode::Sextend {
-            assert!(out_bits == 32 || out_bits == 64);
-            let sign_extend = op == Opcode::Sextend;
-            let inner_ty = ctx.input_ty(insn, 0);
-            let inner_bits = ty_bits(inner_ty);
-            assert!(inner_bits < out_bits);
-            let extendop = match (sign_extend, inner_bits) {
-                (true, 1) => ExtendOp::SXTB,
-                (false, 1) => ExtendOp::UXTB,
```

Comment:
I dropped these, as there is no `i1` type and extension of `b1` should use `bextend` and not `uextend` or `sextend`.